src/app/add-member-form/add-member-form.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { Validators } from '@angular/forms';
+
+
+@Component({
+  selector: 'app-add-member-form',
```

Comment:
```suggestion
  selector: 'add-member-form',
```

optional you could remove the default `app` designation here. It's just noisy.